baseplate/server/thrift.py

```diff
@@ -59,7 +59,7 @@ def make_server(server_config: Dict[str, str], listener: socket.socket, app: Any
         server_config,
         {
             "max_concurrency": config.Integer,
-            "stop_timeout": config.Optional(config.Integer, default=0),
+            "stop_timeout": config.Optional(config.Integer, default=10),
```

Comment:
These should probably be `config.Timespan` as a config value of `15 seconds` doesn't work with `config.Int`.

Reply:
thanks! definitely want to be consistent on using timespans. updated.

Reply:
If we use a value like `15` right now, that will be backwards compatible with the update, correct?

Reply:
correct

Reply:
Perfecto, thanks!